ecl/hthor/hthorkey.cpp

```diff
@@ -1253,6 +1255,8 @@ const void *CHThorIndexNormalizeActivity::nextRow()
                 if (ret)
                     return ret;
             }
+            else
+                callback.finishedRow(); // first could filter
```

Comment:
>I think the result of this change is identical. Is there a case when it is called at a different time?

it is mainly that it wasn't called after next()  (and I had assumed blob could be loaded by first() + next()), and with the added call within createNextRow it would have called finishedRow twice needlessly, so the removal of the finishedRow call on line 1237, and the explicit call to it on the !expanding is to replace that.

Now that I know it is not needed after next() then I can change it.